app_dart/lib/src/model/ci_yaml/target.dart

```diff
@@ -208,6 +210,9 @@ class Target {
       return value.replaceAll('\\n', '\n');
     } else if (int.tryParse(value) != null) {
       return int.parse(value);
+    } else if (double.tryParse(value) != null) {
```

Comment:
Is the comment right above relevant, that the changes need to be also made in the infra repo?
https://github.com/flutter/cocoon/blob/4369cde1522bf88dfc2b6f99fb62c8a45108ae3c/app_dart/lib/src/model/ci_yaml/target.dart#L194-L197

Reply:
I'll investigate tomorrow and see

Reply:
I think I need to update `_parse_attribute` from there, but I need to grok python now :)

Reply:
starlark does not support `float`; so if I do nothing on Luci - this will get parsed as a string (since `isdigit()` will return false).

I do not expect to do anything with this new flag on Luci and only on Cocoon; so I don't think I need to do this at all.

Reply:
Maybe add a comment then, like "double is not currently implemented in LUCI" or something similar.

Reply:
done